lib/config/rules/react.js

```diff
@@ -0,0 +1,27 @@
+// tslint-react
+// Lint rules related to React & JSX for TSLint.
+// Reference https://github.com/palantir/tslint-react
+
+module.exports = {
+  // Enforces a consistent style for multiline JSX elements which promotes ease of
+  // editing via line-wise manipulations as well as maintainabilty via small diffs when changes are made.
+  'jsx-alignment': true,
+  // Allows blacklisting of props in JSX with an optional explanatory message in the reported failure. <FEEDBACK>
+  'jsx-ban-props': false,
+  // Requires or bans spaces between curly brace characters in JSX; Rule options: ["always", "never"]
+  'jsx-curly-spacing': ['always'],
+  // Creating new anonymous functions (with either the function syntax or ES2015 arrow syntax) inside the render call stack works against pure component rendering.
+  // When doing an equality check between two lambdas, React will always consider them unequal values and force the component to re-render more often than necessary. <FEEDBACK>
+  'jsx-no-lambda': false,
```

Comment:
It doesn't :/  There's an [open issue to implement a `jsx-no-bind` rule](https://github.com/palantir/tslint-react/issues/44).